ios/FluentUI/Resources/Localization/StringKeys.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Foundation
+
+/// Keys for getting localized strings.
+@objc(MSFStringKeys)
+public class StringKeys: NSObject {
```

Comment:
I disagree on the `N/A` for binary size: adding a new public class will definitely have a measurable impact. Let's get it recorded just for reference.

Reply:
updated